src/content/wiki/usage.mdx

```diff
@@ -127,11 +127,13 @@ line and immediately exit.
 
 ### --check
 
-**Type:** `boolean`
+**Type:** `string`
 
 Perform a "diagnosis report" where the results of the diagnosis are written to a
```

Comment:
While we are here, may as well change this to read something along the lines of:

> Perform a "diagnosis report" where the results of the diagnosis are written to the \[logpath\](/wiki/faq/#where-can-i-find-the-log-file).

Reply:
thanks and done; please take a look if it works for you.